future_challenges_for_RSEng.tex

```diff
@@ -278,14 +279,39 @@ \subsection{Definition and Evolution of RSEs}
 % European quantum computing: https://digital-strategy.ec.europa.eu/en/policies/quantum
 \end{FramedParagraphWithFootnotes}
 
+\begin{FramedParagraphWithFootnotes}
+GC-NOTE: With longer-term funding and more trust to pick up longer term projects,
+some RSEs can focus on code modernization, cleanup, and modularization,
+which can pay off in faster and larger development in the future.
+This is not possible by short-term funded solo RSEs.
+\end{FramedParagraphWithFootnotes}
 
 \subsection{Education, and changes in the educational system}
+
+\begin{FramedParagraphWithFootnotes}
 FG-NOTE: I wonder if we need a separation between education of RSEs and changes in the education of the broader society.
+\end{FramedParagraphWithFootnotes}
+
+\begin{FramedParagraphWithFootnotes}
+GC-NOTE: Digital competencies in the general population continue to increase, making jumping to software engineering easier than before.
```

Comment:
is that really true? I don't know. Certainly digital literacy was mostly new to my SpaceMed master students. I think we would need some reference to back this up.